resources/views/metadata-title.blade.php

```diff
@@ -0,0 +1,3 @@
+{{-- Uses a regular `echo` expression since blade syntax creates unexpected HTML
+entities like `&#039;` that doesnt work correctly when shared for example. --}}
+<title><?php echo htmlentities(trim($slot)) ?></title>
```

Comment:
Can you explain why the `title` tag is set with this? the issue appears in the meta title, not the regular title tag 🤔 
![image](https://user-images.githubusercontent.com/35610748/130438181-ed566ed6-b669-41f6-99ab-7e2f85e5c0ba.png)

Reply:
@ItsANameToo mm... the problem is (potentially) in all the meta (including regular `title`) (Ill use the same solution to the rest of the metatags)

The reason: we add the meta data like this (`View::yieldContent('meta-title', trans('metatags.home.title'))`) and the we use the blade `{{ }}` directive to then write the content `<meta name="description" content="{{ trim($description) }}" />` and the problem is that the blade directive ( `{{ }}` ) encodes the chars.

That's an alternative to the regular `@yield` directive and I didn't add those components but I am pretty sure the reason was to trim the texts and extract the components.